.gitignore

```diff
@@ -15,3 +15,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+example/.powenv
```

Comment:
As their's a `example/.powenv.sample` showing how to setup is environment variables with Pow I think not having this could quickly leak credentials to Git :/

Why don't you like it?